README.md

```diff
@@ -15,13 +15,14 @@ Support for:
 * Filled: 1-100y5
 * Staggered: 1-100:3 (1-100x3, 1-100x2, 1-100)
 * Negative frame numbers: -10-100
+* Subframes: 1001-1066x0.25
```

Comment:
Can you also apply all of these README updates to the module docstring in `fileseq/__init__.py` so that it shows up in the sphinx API docs?